Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM hashicorp/packer:1.6.5
 
-ARG WINDOWS_UPDATE_VERSION=0.10.1
+ARG WINDOWS_UPDATE_VERSION=v0.14.0
```

Comment:
```suggestion
ARG WINDOWS_UPDATE_VERSION=0.14.0
```

Reply:
The updatecli manifest didn't filtered the "v", fixed in the previous commit but sadly not taken in account in this PR.